### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by tockn**

### CODE REVIEW

File: daemon/images/image.go [hunk 1]
```diff
@@ -161,12 +161,12 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 		}
 		p := *options.Platform
 		// Note that `platforms.Only` will fuzzy match this for us
-		// For example: an armv6 image will run just fine an an armv7 CPU, without emulation or anything.
+		// For example: an armv6 image will run just fine an armv7 CPU, without emulation or anything.
```
Comment (thaJeztah) [inline daemon/images/image.go:164]: This probably had to be "on an armv7 CPU"

```
// For example: an armv6 image will run just fine on an armv7 CPU, without emulation or anything.
```

Comment (tockn) [reply daemon/images/image.go]: thanks! fixed!

**Comment by samuelkarp** [comment]
Can you [sign off your commit](https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work)?

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**